File: kinotic-domain/src/main/java/org/kinotic/domain/api/model/WatchedType.java
package org.kinotic.domain.api.model;

/**
 * The kinds of {@link Watched} record, the first half of a {@link WatchedParent}.
 */
public enum WatchedType {
    WORKLOAD,
    PROJECT_DEPLOYMENT,
    MICROSERVICE_DEPLOYMENT,
    UI_DEPLOYMENT,
    VM_NODE,
    JOB_RUN
}

File: kinotic-js/workspace/packages/management-api/src/api/model/WatchedType.ts
/**
 * The kinds of watched record, the first half of a {@link WatchedParent}.
 */
export enum WatchedType {
    WORKLOAD = 'WORKLOAD',
    PROJECT_DEPLOYMENT = 'PROJECT_DEPLOYMENT',
    MICROSERVICE_DEPLOYMENT = 'MICROSERVICE_DEPLOYMENT',
    UI_DEPLOYMENT = 'UI_DEPLOYMENT',
    VM_NODE = 'VM_NODE',
    JOB_RUN = 'JOB_RUN'
}
